resources/assets/js/features/user/components/Register/Form.js

```diff
@@ -72,6 +95,13 @@ class Form extends React.Component {
                                 completeTrip();
                                 browserHistory.push('/trips');
                             });
+
+                        } else {
+                            browserHistory.push('/dashboard');
```

Comment:
if user doesn't have pending trip and doesn't have pending booking he will be navigated twice to dashboard